acrobat/blocks/verb-widget/verb-widget.js

```diff
@@ -147,32 +136,43 @@ export default async function init(element) {
   });
 
   // Errors, Analytics & Logging
+  const handleError = (str) => {
+    errorState.classList.add('verb-error');
+    errorState.classList.remove('hide');
+    errorStateText.textContent = str;
+
+    setTimeout(() => {
+      errorState.classList.remove('verb-error');
+      errorState.classList.add('hide');
+    }, 5000);
+  };
+
   window.addEventListener('unity:show-error-toast', (e) => {
     console.log(`⛔️ Error Code - ${e.detail?.code}`);
 
     if (e.detail?.code === 'only_accept_one_file') {
-      handleError(errorState, errorStateText, 'verb-widget-error-multi');
+      handleError(e.detail?.message);
       verbAnalytics('error', VERB);
     }
 
     if (e.detail?.code === 'unsupported_type') {
-      handleError(errorState, errorStateText, 'verb-widget-error-unsupported');
+      handleError(e.detail?.message);
       verbAnalytics('error:unsupported_type', VERB);
     }
 
     if (e.detail?.code === 'empty_file') {
-      handleError(errorState, errorStateText, 'verb-widget-error-empty');
+      handleError(e.detail?.message);
       verbAnalytics('error:empty_file', VERB);
     }
 
     // Code may be wrong. should be 'file_too_large'
     if (e.detail?.code === 'file_too_largempty_file') {
-      handleError(errorState, errorStateText, 'verb-widget-error-large', LIMITS[VERB].maxFileSizeFriendly);
+      handleError(e.detail?.message);
       verbAnalytics('error', VERB);
     }
 
     if (e.detail?.code === 'max_page_count') {
-      handleError(errorState, errorStateText, 'verb-widget-error-max', LIMITS[VERB].maxNumFiles);
+      handleError(e.detail?.message);
       verbAnalytics('error:max_page_count', VERB);
     }
```

Comment:
The codes that we send will be based off `title.single.code` in this sheet: [fillsign.xlsx](https://adobe.sharepoint.com/:x:/r/sites/AdotcomEnterprisePersonalization/Shared%20Documents/2.%20Platform/Product%20Area%20Documents/Feds%20Platform%20Capabilities/Project%20Unity/error%20handling/[fillsign.xlsx](https://adobe.sharepoint.com/:x:/r/sites/AdotcomEnterprisePersonalization/Shared%20Documents/2.%20Platform/Product%20Area%20Documents/Feds%20Platform%20Capabilities/Project%20Unity/error%20handling/fillsign.xlsx?d=w118b19a1af214fd3a7349b473a43bff3&csf=1&web=1&e=IIbcEx)?d=w118b19a1af214fd3a7349b473a43bff3&csf=1&web=1&e=IIbcEx). Could we update these equality checks to `includes()` instead?

Reply:
maybe @amitbikram can confirm reasoning behind using this specific format (ie. `verb_upload_error...`)?

Reply:
Sorry I will need `error` in the prefix for the code to distinguish on unity end, but `verb_upload` seems extraneous.